unblob/file_utils.py

```diff
@@ -426,6 +427,9 @@ def has_root_permissions(self):
     def _fs_path(self, path: Path) -> _FSPath:
         return _FSPath(root=self.root, path=path)
 
+    def _ensure_parent_dir(self, path: Path):
+        path.parent.mkdir(parents=True, exist_ok=True)
+
     def get_checked_path(self, path: Path, path_use_description: str) -> Optional[Path]:
```

Comment:
I know it was in the previous PR, but this function for me is more like a get_absolute_path (which also checks for traversal). It is not just a checking, rather a conversion with a check

Reply:
This is a function that does 2 common things, because it is removing a common boilerplate (smelly I agree). However the emphasis is definitely on the checking side, so `get_absolute_path` would be a very misleading name, as there is no explicit intention to return an *absolute* path, the goal is to return an *actionable* path, or `None` if it is not possible. Returning an absolute path is the easiest implementation of the actionable path, but being absolute is not an important property, and could be replaced with a working directory relative actionable path as well.

Possible improvements:
- make it protected/private, as it is not something to use externally
- change name to `_get_actionable_path`